pkg/cstorvolumeclaim/v1alpha1/build.go

```diff
@@ -303,15 +303,27 @@ func (b *Builder) WithCapacity(capacity string) *Builder {
 	return b.WithCapacityQty(resCapacity)
 }
 
-// WithCapacityQty sets Capacity of CStorVOlumeClaim
+// WithCapacityQty sets Capacity of CStorVolumeClaim
 func (b *Builder) WithCapacityQty(resCapacity resource.Quantity) *Builder {
-	resourceList := metav1.ResourceList{
```

Comment:
there were discussions around this.. can this get approved from @AmitKumarDas or @mittachaitu ?

Reply:
This also good only intentions we shouldn't maintain capacity as a string but we can have capacity as `resource.Quantity` or `corev1.ResourceList`.
maintaining: 
`resource.Quantity`  will be useful only to store storage capacity
`corev1.ResourceList` can have multiple fields as below (depends on usecase) 
```
	// CPU, in cores. (500m = .5 cores)
	ResourceCPU ResourceName = "cpu"
	// Memory, in bytes. (500Gi = 500GiB = 500 * 1024 * 1024 * 1024)
	ResourceMemory ResourceName = "memory"
	// Volume size, in bytes (e,g. 5Gi = 5GiB = 5 * 1024 * 1024 * 1024)
	ResourceStorage ResourceName = "storage"
	// Local ephemeral storage, in bytes. (500Gi = 500GiB = 500 * 1024 * 1024 * 1024)
	// The resource name for ResourceEphemeralStorage is alpha and it can change across releases.
	ResourceEphemeralStorage ResourceName = "ephemeral-storage"
```